config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -274,12 +276,13 @@ spec:
                     More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
                   type: string
                 kubernetesVersion:
-                  description: KubernetesVersion is the target version of the control
-                    plane.
+                  description: "KubernetesVersion is the target version of the control
+                    plane. \n NB: This value defaults to the Machine object spec.kuberentesVersion"
```

Comment:
This \n looks weird here

Reply:
That's because of the multi-line comment, this happens everywhere else as well I think.

Reply:
It seems it only happens if you leave empty new lines in a comment, very strange.